holidays/countries/turks_and_caicos_islands.py

```
@@ -0,0 +1,107 @@
+#
+#  Authors: Vacanza Team and individual contributors (see CONTRIBUTORS file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+from gettext import gettext as tr
+
+from holidays.groups import ChristianHolidays, InternationalHolidays
+from holidays.calendars.gregorian import JUN
+from holidays.constants import MON
```

Ruff (0.8.2)

`holidays.constants.MON` imported but unused

Remove unused import: `holidays.constants.MON`

(F401)